datalog/examples/printlog/printlog.cpp

```diff
@@ -9,11 +9,10 @@
 #include <fmt/chrono.h>
 #include <fmt/format.h>
 #include <fmt/ranges.h>
-
-#include "wpi/DataLogReader.h"
-#include "wpi/DenseMap.h"
-#include "wpi/MemoryBuffer.h"
-#include "wpi/print.h"
+#include <wpi/DenseMap.h>
+#include <wpi/MemoryBuffer.h>
+#include <wpi/datalog/DataLogReader.h>
```

Comment:
The wpi/datalog include being in <> indicates a .styleguide needs to be added to the datalog/ level (wpi/datalog includes should be in "", other wpi/ includes should be in <>).